content/applications/finance/fiscal_localizations/uruguay.rst

```diff
@@ -0,0 +1,321 @@
+=======
+Uruguay
+=======
+
+.. |DGI| replace:: :abbr:`DGI (Dirección General Impositiva)`
+.. |EDI| replace:: :abbr:`EDI (Electronic Data Interchange)`
+
+Introduction
+============
+
+With the Uruguayan localization, you can generate electronic documents with its XML, fiscal folio,
+electronic signature and connection to tax authority Dirección General Impositiva (DGI) through
+Uruware.
+
+The supported documents are:
+
+- Invoices, credit notes, debit notes;
+- e-ticket e-ticket credit notes, e-ticket debit notes;
+- Export e-invoice, export e-credit notes, export e-debit notes.
```

Comment:
```suggestion
- Export e-Invoice, Export e-Invoice Credit Note, Export e-Invoice Debit Note.
```

Reply:
Added this comments to be consistent on how the documents are named in the database.